.github/workflows/riscV.yml

```diff
@@ -0,0 +1,46 @@
+name: RISC-V
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-sp1:
+    runs-on: ubuntu-latest
+    needs: [cargo-fmt, cargo-clippy]
```

Comment:
looks like the risc CI's are now not running?

trying removing the needs lines for both of them. I am guessing it is due to these jobs not being in the same file node